src/prosper/feature.js

```diff
@@ -71,3 +71,19 @@ var countChar = function(word, character) {
 var countBs2 = function(word) {
 	return countChar(word, "B");										//call the function "countChar" with string "B" as the second input
 }
+
+
+var sequence = function(n) {
```

Comment:
A more descriptive parameter name is recommended. This will improve the clarity of code.